### PR TITLE
Fetch notes Use Case

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

File: api/test/use-cases/FetchNotes.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  it('can sort the documents by date descending', async () => {
+    const expectedNotes = {
+      notes: [].concat(notesFromA, notesFromB).sort((a, b) => b.date - a.date)
+    };
+
+    const notes = await fetchNotes();
+
+    expect(notes).toEqual(expect.objectContaining(expectedNotes));
+  });
```
Comment (bjpirt) [inline api/test/use-cases/FetchNotes.test.js]: Might also be worth adding a test for "it should remove duplicate notes from the results"

Comment (mmmmillar) [reply api/test/use-cases/FetchNotes.test.js]: I think that would be testing what "dedupe" does

Comment (mmmmillar) [reply api/test/use-cases/FetchNotes.test.js]: i think that would be testing what "dedupe" does
